operations/mimir-rules-action/README.md

```diff
@@ -11,6 +11,7 @@ This action is configured using environment variables defined in the workflow. T
 | `MIMIR_ADDRESS`              | URL address for the target Mimir cluster                                                                                                                                                                                                                                                           | `false`  | N/A     |
 | `MIMIR_TENANT_ID`            | ID for the desired tenant in the target Mimir cluster. Used as the username under HTTP Basic authentication.                                                                                                                                                                                       | `false`  | N/A     |
 | `MIMIR_API_KEY`              | Optional password that is required for password-protected Mimir clusters. An encrypted [github secret](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets) is recommended. Used as the password under HTTP Basic authentication. | `false`  | N/A     |
+| `MIMIR_AUTH_TOKEN`           | Optional bearer or JWT token that is required for Mimir clusters authenticating by bearer or JWT token. An encrypted [github secret](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets) is recommended.                         | `false`  | N/A     |
```

Comment:
```suggestion
| `MIMIR_AUTH_TOKEN`           | Optional bearer or JWT token that is required for Mimir clusters authenticating by bearer or JWT token. | `false`  | N/A     |
```

I would remove mention of "An encrypted github secret is recommended." because `mimirtool` user cannot control what the Mimir installation (or gateway in front of it) is using.

But I see this is copied from `MIMIR_API_KEY` description, so we can address both later.

Reply:
The reason why the GitHub secret is mentioned here is different. This is the `mimir-rules-action` and it's suggesting to use a GitHub secret to hold this env variable value.